### PR TITLE
Add option to helper to customize others label

### PR DESCRIPTION
**Pull request by elenatorro**
Fix https://github.com/CartoDB/airship/issues/604
Related Airship PR https://github.com/CartoDB/airship/pull/606

Screenshot 2019-05-24 at 18 17 49

### CODE REVIEW

File: cartoframes/viz/helpers/color_category_layer.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from ..layer import Layer
 
 
-def color_category_layer(source, value, top=11, palette='bold', title=''):
+def color_category_layer(source, value, top=11, palette='bold', title='', othersLabel='Others'):
```
Comment (Jesus89) [inline cartoframes/viz/helpers/color_category_layer.py]: I think by now we should use a fixed value in the legend definition

File: cartoframes/viz/helpers/color_category_layer.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def color_category_layer(source, value, top=11, palette='bold', title=''):
                 'color': 'ramp(top(${0}, {1}), {2})'.format(value, top, palette)
             },
             'polygon': {
-                'color': 'opacity(ramp(top(${0}, {1}), {2}),0.9)'.format(value, top, palette)
+                'color': 'opacity(ramp(top(${0}, {1}), {2}), 0.1)'.format(value, top, palette)
```
Comment (Jesus89) [inline cartoframes/viz/helpers/color_category_layer.py]: Restore this to 0.9

File: cartoframes/viz/helpers/color_category_layer.py [hunk 3]
```diff
@@ -27,6 +27,7 @@ def color_category_layer(source, value, top=11, palette='bold', title=''):
             'type': 'basic',
             'ramp': 'color',
             'heading': title or value,
-            'description': ''
+            'description': '',
+            'othersLabel': othersLabel
```
Comment (Jesus89) [inline cartoframes/viz/helpers/color_category_layer.py]: Add 'Others' here

**Comment by elenatorro** [comment]
Great! Yes, I use flake8 but it was misconfigured in the editor, now is working again. I'll merge this once I update Airship legends branch 👍

**Pull request merged**